packages/indicators/src/Builder/helpers.ts

```diff
@@ -55,6 +55,9 @@ export const evaluateFormulaToNumber = (
   return value;
 };
 
+export const isValidDataValues = (value: string | number) =>
```

Comment:
```suggestion
export const isValidIndicatorValue = (value: string | number) =>
```
Small rename suggestion 👍